LoopKit/GlucoseKit/GlucoseStore.swift

```diff
@@ -51,7 +46,7 @@ public enum GlucoseStoreResult<T> {
  */
 public final class GlucoseStore: HealthKitSampleStore {
 
-    /// Notification posted when glucose samples were changed, either via add/replace/delete methods or from HealthKit
+    /// Notification posted when glucose samples were changed, either via direct add or from HealthKit
```

Comment:
[nit] Is this signal changing to only include adds?  Or are replace/delete's still firing this signal?

Reply:
Technically this is sent for *any* change to the GlucoseStore, but changes to the GlucoseStore can only happen on direct add or HealthKit update. There is no way to direct replace or delete glucose samples (only via HealthKit from an external app). The "deletion" that does occur is through purge (either when sample fall off the end of the cache or when an external app adds entries that this cache is also managing) and purging only occurs at the end of a direct add or HealthKit update. So, technically, glucose samples are only changed via direct add or HealthKit update (which may also purge, but that is an internal implementation detail.)

Reply:
I'm just thinking about this from a caller's perspective.  If someone calls `deleteCarbEntry` or `replaceCarbEntry`, will this signal be fired? If so, I think we should document this notification as responding to add, replace, deletes, and adds or deletes from HK.

Reply:
This is the glucose store, not the carb store. A user of the glucose store cannot explicitly replace or delete a glucose entry (and deletes only happen from a purge related to an add or HealthKit operation).

The carb store docs do say "either via add/replace/delete methods or from HealthKit" since a user of the carb store can explicitly replace or delete carbs.

Reply:
Doh! In the wrong pew. :) Sorry, and thanks for pointing out my mistake.